docs/troubleshooting.md

```diff
@@ -14,11 +14,12 @@ FORCE_WRITE=true make generate
 
 ## kssh is slow, but it works
 
-When kssh starts, it has to search every team you are in for a `kssh-client.config` file which specifies the information
-that is needed in order to communicate with the CA chatbot. If you are only in a few teams, this is relatively fast 
-(1-2 seconds for <10 teams) but this can become much slower as the number of teams increases (6 seconds for 100 teams
-in my benchmarks). This complex start up procedure can be avoided by setting a default bot via 
-`kssh --set-default-bot cabotname` which should reduce kssh's startup time considerably. 
+When kssh starts, it has to search the KV store for every team you are in for
```

Comment:
i doubt this is slow anymore even for a lot of teams. i suspect we could rip out the whole `--set-default-bot` construct and flow. it's easier to leave it for now. just a passing thought.

Reply:
maybe i'm missing something, but with this no longer an issue, there's really not much value to having a client config file at all. all of that persistent state isn't nearly as valuable as the added complexity of having some configs sometimes in a file and getting them there and checking for them. the way this thing is used, most people are going to be hitting the same servers and the same bot every time, so if kssh only exposed `kssh --bot yourcabot <rest of command>` in the event you were in multiple teams with ca-bots, you just write your own alias once (e.g. `kssh-prod`) and never think about it again, and then kssh gets the added benefit of being basically stateless (excepting your non-kssh ssh configs). 

not a task for this PR. but the move from kbfs->kvstore seems like it offers some additional opportunities.

Reply:
Removing the local kssh config is a good idea for a future improvement!